CMakeLists.txt

```diff
@@ -0,0 +1,138 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.6)
```

Comment:
I don't know cmake well enough to know what this file is for. Was it an intentional commit?

Reply:
Yeah. I often develop with the CLion IDE, which uses CMake. That file won't allow you to build Soufflé with CMake, but it does allow CLion to provide all the usual helpful functionality of an IDE.